### DISCUSSION TITLE
How do I update the webconfig?

### DISCUSSION THREAD

**Discussion by tomjohnstone** [hrvach/deskhop#123] [Q&A] [answered]
Hi,

I have forked the rc branch and made some changes to the webconfig. I can successfully make a new firmware and deploy it onto my device however I'm not seeing my updated webconfig. Are there additional steps required to deploy a new web config?

Thanks,
Tom

**Comment by tomjohnstone** [comment, marked as answer]
I figured it out for anyone else wanting to know how to rebuild the webconfig:

```
python3 webconfig/render.py
disk/create.sh
cmake --build build
```

**Reply by hrvach** [reply]
Thanks for sharing, updated the README with the instructions. Ideally it should also get linked in the main build script so no additional steps are required.
